app/components/playfair/elements/VerticalGrid.tsx

```diff
@@ -1,22 +1,22 @@
-export default function VerticalGrid(props) {
+export default function VerticalGrid({ xValue, offset, opacity, text }) {
   return (
     <g>
       <line
-        x1={props.xValue + props.offset}
+        x1={xValue + offset}
         y1="3"
-        x2={props.xValue + props.offset}
+        x2={xValue + offset}
         y2="47"
-        opacity={props.opacity ? props.opacity : "0.4"}
+        opacity={opacity?.toString() ? opacity : "0.4"}
```

Comment:
Should this be a nullish coalescing? 
```suggestion
        opacity={opacity ?? "0.4"}
```